From 49dc48901da4dbc2f05eb77c4ceff753f210f64b Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Fri, 15 Dec 2023 12:03:59 +0100
Subject: [PATCH 0745/1141] arm64: Kconfig: select STM32MP_EXTI on STM32
 platforms

Use the new config flag to build the correct driver extracted from
the old code.

Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: I435e4ea5b2449b33e901c38ed154c649e995d331
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/347754
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Alexandre TORGUE <alexandre.torgue@foss.st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 arch/arm64/Kconfig.platforms | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/Kconfig.platforms b/arch/arm64/Kconfig.platforms
index b771b692e143..63c998178e8a 100644
--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -285,7 +285,7 @@ config ARCH_STM32
 	select GPIOLIB
 	select PINCTRL
 	select RESET_CONTROLLER
-	select STM32_EXTI
+	select STM32MP_EXTI
 	select COMMON_CLK_STM32MP257
 	select RESET_STM32MP1
 	select ARM_SMC_MBOX
-- 
2.39.2

